docs/get-started.md

```diff
@@ -157,7 +158,23 @@ There is a quick fix however:
 2. In the Project panel, navigate to **Assets** > **Fps** > **Art** > **Materials**.
 3. Right click on `Source_Shaders` and press Reimport.
 
-<%(#Expandable title="Reimport shaders")%><img src="{{assetRoot}}assets/shader-fix.jpg" style="margin: 0 auto; display: block;" /><%(/Expandable)%>
+<img src="{{assetRoot}}assets/shader-fix.jpg" style="margin: 0 auto; display: block;" />
```

Comment:
Is there a raeson to use this HTML-like syntax?

Reply:
I'm not sure - I'm just using whatever was already in the repo

@samc-improbable ?

Reply:
can use `![alt](url)` i think

Reply:
yup I've used ![alt](url) before, just didn't want to change unless there was a specific reason for putting it like that

Reply:
reasoning for html style was to ensure images are aligned properly - not so easy to do in MD

Reply:
Yeah it keeps it centered